docker/generic/start_proxy.py

```diff
@@ -150,12 +150,12 @@ def make_argparser():
         HTTP/2 secure connections on listener_port. Requires the certificate and
         key files "server.crt" and "server.key" within this path.''')
 
-    parser.add_argument('--ssl_client_cert_path', default=None, help='''
+    parser.add_argument('--ssl_backend_client_cert_path', default=None, help='''
```

Comment:
What about multiple backends?   Each backend has its own ssl cert files or root files?  Do we want to support that?

It is OK we don't support it now, but we may need to have a plan that it is easy to support it in the future, especially not introduce a breaking changes.

Reply:
I see. We can follow the same approach as we decided for the `--backend` flag:
1. For now, keep the `--ssl_backend_client_cert_path` flag that applies to all backends.
2. If we need to support multiple backends in the future, then add it into the service config. Have this flag override it if the field is empty.